components/ProjectList/index.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import styled from "styled-components";
+import Card from "../Card";
+import useSWR from "swr";
+
+export default function ProjectList() {
+  const {
+    data: projects,
+    isLoading,
+    error,
+  } = useSWR("/api/projects", {
+    fallbackData: [],
+  });
+
+  if (error) return <p>An error has occurred.</p>;
+  if (!projects || isLoading) return <p>Projects are loading</p>;
+  return (
+    <>
+      {projects.map((project, index) => (
+        <Card key={index} project={project} />
```

Comment:
Please don't ever use the index of an array as the `key` value. This is a very unreliable method and is bound to create unnecessary re-renders.